### PR TITLE
Add new System field to ClearImpound and Thargoids to Statistics

### PR DESCRIPTION
**Pull request by Eahlstan**

### CODE REVIEW

File: EliteDangerous/JournalEvents/JournalBountyCrimesBondsFines.cs [hunk 1]
```diff
@@ -460,17 +460,19 @@ public JournalClearImpound(JObject evt) : base(evt, JournalTypeEnum.ClearImpound
...
         public string ShipType_Localised { get; set; }
         public ulong ShipId { get; set; }
         public long ShipMarketID { get; set; }
         public long MarketID { get; set; }
+        public string System { get; set; }  //patch 17, so may be null
 
         public override void FillInformation(out string info, out string detailed)
         {
-            info = BaseUtils.FieldBuilder.Build("Ship: ".T(EDCTx.JournalEntry_Ship), ShipType_Localised);
+            info = BaseUtils.FieldBuilder.Build("Ship: ".T(EDCTx.JournalEntry_Ship), ShipType_Localised, "In system: ".T(EDCTx.JournalClearImpound_InSystem), System);
```
Comment (robbyxp1) [inline EliteDangerous/JournalEvents/JournalBountyCrimesBondsFines.cs:475]: Missing ID

File: EliteDangerous/EliteDangerous/Translations.cs [hunk 2]
```diff
@@ -477,8 +478,16 @@ internal enum EDCTx
...
         ExobiologyClass_OrganicGenus, // Unique Genus Data Logged: ;;N0
         ExobiologyClass_OrganicSpecies, // Unique Species Data Logged: ;;N0
-
-    JournalNavRoute_Jumps, // "{0} jumps: "
+        ThargoidsClass_EncounterWakes, //Thargoid wakes scanned: ;;N0
+        ThargoidsClass_EncounterKilled, //Thargoids killed: ;;N0
+        ThargoidsClass_EncounterImprint, //Thargoid structures: ;;N0
+        ThargoidsClass_EncounterTotal, //Total encounters: ;;N0
+        ThargoidsClass_LastSystem, //Last seen in: 
+        ThargoidClass_LastTime, //Last seen on:
```
Comment (robbyxp1) [inline EliteDangerous/EliteDangerous/Translations.cs:486]: missing s!

File: EliteDangerous/JournalEvents/translation-example-je.tlp [hunk 3]
```diff
@@ -861,6 +861,17 @@ SECTION ExobiologyClass
 .OrganicGenus: "Unique Genus Data Logged: ;;N0" @
 .OrganicSpecies: "Unique Species Data Logged: ;;N0" @
 
+SECTION ThargoidsClass
+
+.EncounterWakes: "Thargoid wakes scanned: ;;N0" @
+.EncounterKilled, "Thargoids killed: ;;N0" @
+.EncounterImprint, "Thargoid structures: ;;N0" @
+.EncounterTotal, "Total encounters: ;;N0" @
```
Comment (robbyxp1) [inline EliteDangerous/JournalEvents/translation-example-je.tlp:869]: Colon not comma

Comment (Eahlstan) [reply EliteDangerous/JournalEvents/translation-example-je.tlp:869]: Stupid me copying things without changing it properly, sorry.
Wonder why I stopped after the first...

File: EliteDangerous/JournalEvents/JournalBountyCrimesBondsFines.cs (hunk 1 above)
Comment (Eahlstan) [reply EliteDangerous/JournalEvents/JournalBountyCrimesBondsFines.cs:475]: You can see it was in there in the first commit: https://github.com/EDDiscovery/EliteDangerousCore/pull/133/commits/9e4ffe9d10f7d4c2feaddfda10d1d0f2e38d6fdb
You can steal it back from there.

**Pull request merged**